src/js/actions/FriendActions.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
   },
 
   emailBallotData: function ( email_address_array, first_name_array, last_name_array, email_addresses,
-                              invitation_message, ballot_link, sender_email_address) {
+                              invitation_message, ballot_link, sender_email_address, verification_email_sent) {
```

Comment:
Might be easier to read if this is separated into an interface that is passed in.

Reply:
Thank you @pertrai1. What would be an example of separating this into an interface?

Reply:
@DaleMcGrew If there is some type of utility file of some sorts that has generic exports, you can have one such as:

Utilities.js (example name)
```
const userData = {
    email_address_array,
    first_name_array,
    etc...
}

export userData
```

FriendAction.js
```
import { userData } from 'Utilities'

emailBallotData({...userData} = {}) {
    // body here 
    // userData gives the the object that you can then use in the body
}
```

This is not an exact replica of what you have. Give it a shot or I can help if needed.